app/components/post_list/post_list.android.js

```diff
@@ -20,20 +21,28 @@ export default class PostList extends PostListBase {
         this.state = {
             refreshing: false,
             managedConfig: {},
-            dataSource: new DataSource(props.postIds, this.keyExtractor),
+            dataSource: new DataSource(props.postIds, keyExtractor),
+            lastPostIndex: getLastPostIndex(this.props.postIds),
+            postIds: this.props.postIds,
```

Comment:
@saturninoabril in this PR #2761 we are removing this file, how do you want to handle it?

Reply:
PR rebase and updated.  What I did was instead of deriving a state, I just pass `lastPostIndex` as props into it.  
To apply it for Android only, I made it like,
```javascript
<PostList
    lastPostIndex={Platform.OS === 'android' ? getLastPostIndex(visiblePostIds) : -1}
    ...
/>
```